commands.go

```diff
@@ -147,6 +149,25 @@ var commandThrow = cli.Command{
 	},
 }
 
+var commandMetrics = cli.Command{
+	Name:      "metrics",
+	Usage:     "Fetch metric values",
+	ArgsUsage: "[--host | -h <hostId>] [--service | -s <service>] stdin --from int --to int",
```

Comment:
It looks like `commandMetrics` does not take `stdin` as input, but does `--name | -n <metricName>` .